core/templates/dev/head/pages/landing/landing_page_student.html

```diff
@@ -0,0 +1,414 @@
+        <h1 class="oppia-student-landing-h1" style="color: #242424"><[getRowHeading(2)]></h1>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-sm-6 col-sm-push-6 oppia-student-landing-image-div">
+              <img ng-src="<[getRowImageUrl(2)]>" class="oppia-student-landing-image" style="width: 60%" alt="">
+            </div>
+            <div class="col-sm-6 col-sm-pull-6">
+              <h2 class="oppia-student-landing-h2" style="color: #242424"><[getRowParagraph(2, 1)]>
+                <br>
+                <br>
+                <[getRowParagraph(2, 2)]>
+              </h2>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="oppia-student-landing-section text-center" style="background-color: #429488">
+      <div class="oppia-student-landing-section-inner">
+        <h1 class="oppia-student-landing-h1" style="color: #FFFFFF"><[getRowHeading(3)]></h1>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-sm-6 col-sm-push-6 oppia-student-landing-image-div">
+              <img ng-src="<[getRowImageUrl(3)]>" class="oppia-student-landing-image" alt="">
+            </div>
+            <div class="col-sm-6 col-sm-pull-6">
+              <h2 class="oppia-student-landing-h2" style="color: #FFFFFF"><[getRowParagraph(3, 1)]>
+                <br>
+                <br>
+                <[getRowParagraph(3, 2)]>
+              </h2>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="oppia-student-landing-section text-center">
+      <div class="oppia-student-landing-section-inner">
+        <h1 class="oppia-student-landing-h1" style="color: #242424; text-align: right; padding-right: 80px"><[getRowHeading(4)]></h1>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-sm-6 oppia-student-landing-image-div">
+              <img ng-src="<[getRowImageUrl(4)]>" class="oppia-student-landing-image" alt="">
+            </div>
+            <div class="col-sm-6">
+              <h2 class="oppia-student-landing-h2" style="color: #242424; text-align: right; right: 80px"><[getRowParagraph(4, 1)]>
+                <br>
+                <br>
+                <[getRowParagraph(4, 2)]>
+              </h2>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="oppia-student-landing-section text-center" style="background-color: #afd2eb">
+      <div class="oppia-student-landing-section-inner">
+        <h1 class="oppia-student-landing-h1" style="text-align: center; padding-left: 0px">Start your learning today</h1>
+        <h2 class="oppia-student-landing-h2 oppia-student-landing-centered-h2" style="text-align: center; width: 50%;">Fractions are important to learn, and with our lessons you can improve a lot! Matthew needs your help!</h2>
+        <div style="position: absolute; top: 115px">
```

Comment:
@seanlip (Line 188) can we remove the last sentence with something generic? (I didn't find any connection between these two sentences.)

Reply:
I don't understand your question, sorry. What exactly are you proposing to do?

Reply:
On line 188 we have `
Fractions are important to learn, and with our lessons you can improve a lot! Matthew needs your help!` and I wanted to make this line more generic such that we can use this with all landing pages. So I was asking to remove `Matthew needs your help!`.

Reply:
Let's keep this customized for now until we're sure we need the same text across multiple landing pages. It's hard to generalize based on one example.